### PR TITLE
add FileHistoryCache.get() counters for hits/misses

### PR DESCRIPTION
**Pull request by vladak**
No project info to avoid leakage.

### CODE REVIEW

File: opengrok-indexer/src/test/java/org/opengrok/indexer/index/IndexerMainTest.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 import static org.junit.Assert.assertFalse;
 
-public class IndexerMainTest {
+public class  IndexerMainTest {
```
Comment (ahornace) [inline opengrok-indexer/src/test/java/org/opengrok/indexer/index/IndexerMainTest.java:37]: Why the additional space? Maybe a mistake?

Comment (vladak) [reply opengrok-indexer/src/test/java/org/opengrok/indexer/index/IndexerMainTest.java]: yep, inadvertent modification.

File: opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java [hunk 2]
```diff
@@ -174,7 +180,17 @@ protected Expression instantiate(Object oldInstance, Encoder out) {
...
-        // nothing to do
+        MeterRegistry meterRegistry = Metrics.getRegistry();
+        if (meterRegistry != null) {
+            fileHistoryCacheHits = Counter.builder("filehistorycache.history.get").
+                    description("file history cache hits").
+                    tag("what", "hits").
+                    register(meterRegistry);
+            fileHistoryCacheMisses = Counter.builder("filehistorycache.history.get").
+                    description("file history cache misses").
+                    tag("what", "hits").
```
Comment (ahornace) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java:191]: Should not this be a "miss"?

Comment (vladak) [reply opengrok-indexer/src/main/java/org/opengrok/indexer/history/FileHistoryCache.java:191]: oops, thanks !

**Comment by vladak** [comment]
thanks @ahornace !

**Pull request merged**